config/routes.rb

```diff
@@ -12,9 +16,10 @@
       end
       resources :manifests, only: [:create]
       resources :method_codes, only: [:index]
-      patch 'manifests', to: 'manifests#update'
-      get 'manifest', to: 'manifests#show'
+      patch 'manifests/:id', to: 'manifests#update'
+      put 'manifests/:id', to: 'manifests#update'
       get 'manifests/search', to: 'manifests#search'
+      get 'manifests/:id', to: 'manifests#show'
```

Comment:
thanks, yes I tried that earlier in my dev area but because I hadn't also moved the `get /manifests/search` before it, tests were failing. Fixed now in 55a2b1d